examples/05-plotting/02-solution_combination.py

```diff
@@ -0,0 +1,88 @@
+"""
+.. _solution_combination:
+
+Loadcase combination for principal stress and show max/min label.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
@ayush-kumar-423 this line should be as long as the title
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Thanks @cbellot000, done!